sql/moz-fx-data-shared-prod/telemetry_derived/newtab_merino_priors_v1/query.sql

```diff
@@ -1,74 +1,62 @@
-    UNNEST(events) AS event,
-    params
+WITH params AS (
+  SELECT
+    DATE(TIMESTAMP_TRUNC(CURRENT_TIMESTAMP(), DAY) - INTERVAL 1 DAY) AS end_date,
+    DATE(TIMESTAMP_TRUNC(CURRENT_TIMESTAMP(), DAY) - INTERVAL 7 DAY) AS start_date,
+    DATE(TIMESTAMP_TRUNC(CURRENT_TIMESTAMP(), DAY) - INTERVAL 8 DAY) AS start_date_items
+),
+corpus_items AS (
+  SELECT DISTINCT
+    corpus_item_id
+  FROM
+    `moz-fx-data-shared-prod.snowflake_migration_derived.corpus_items_current_v1`
   WHERE
-    submission_timestamp >= params.start_timestamp
-    AND submission_timestamp < params.end_timestamp
-    AND event.category = 'pocket'
-    AND event.name IN ('impression', 'click')
-    AND mozfun.map.get_key(event.extra, 'scheduled_corpus_item_id') IS NOT NULL
-    AND SAFE_CAST(mozfun.map.get_key(event.extra, 'recommended_at') AS INT64) IS NOT NULL
-    AND SAFE_CAST(mozfun.map.get_key(event.extra, 'recommended_at') AS INT64)
-    BETWEEN 946684800000
-    AND UNIX_MILLIS(
-      CURRENT_TIMESTAMP()
-    )  -- This ensures recommended_at is between Jan 1, 2000, and the current time to remain within BQ limits for dates
-),
--- Flatten events and filter relevant data
-flattened_newtab_events AS (
-  SELECT
-    *
-  FROM
-    newtab_events,
-    params
+    DATE(corpus_item_updated_at) > (SELECT start_date_items FROM params)
+),
+base AS (
+  SELECT
+    corpus_item_id,
+    country AS region,    -- rename here
+    impression_count,
+    click_count
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_derived.newtab_content_items_daily_combined_v1`
   WHERE
-    TIMESTAMP_MILLIS(recommended_at) >= params.start_timestamp
-    AND TIMESTAMP_MILLIS(recommended_at) < params.end_timestamp
+    corpus_item_id IS NOT NULL
+    AND country IS NOT NULL
+    AND submission_date
+    BETWEEN (SELECT start_date FROM params)
+    AND (SELECT end_date FROM params)
 ),
--- Aggregate events by scheduled_corpus_item_id and region
+  -- Keep only items that exist in corpus_items and aggregate by item/region
 aggregated_events AS (
   SELECT
-    scheduled_corpus_item_id,
-    region,
-    SUM(IF(event_name = 'impression', 1, 0)) AS impression_count,
-    SUM(IF(event_name = 'click', 1, 0)) AS click_count
+    b.corpus_item_id,
+    b.region,
+    SUM(b.impression_count) AS impression_count,
+    SUM(b.click_count) AS click_count
   FROM
-    flattened_newtab_events
+    base b
+  INNER JOIN
+    corpus_items c
+    ON b.corpus_item_id = c.corpus_item_id
   GROUP BY
-    scheduled_corpus_item_id,
-    region
+    b.corpus_item_id,
+    b.region
 ),
--- Calculate CTR per scheduled_corpus_item_id and region
+  -- Calculate CTR per corpus_item_id and region
 per_region_ctr AS (
   SELECT
-    scheduled_corpus_item_id,
+    corpus_item_id,
     region,
     SAFE_DIVIDE(click_count, impression_count) AS ctr,
     impression_count,
     click_count
   FROM
     aggregated_events
   WHERE
-    impression_count > 0
+    impression_count > 2000
```

Comment:
Could you add a comment about how this value was derived, in case someone wants to recompute it in the future?